src/client/components/button-group/button-group.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <ul class="button-group inline-list">
+    <li class="button-group__item" v-for="item in items" :key="item.slug">
+      <smart-link
+        :item="item"
+        :class="{
+          'button': item.theme === 'button',
+          'button button--primary': item.theme === 'primary button'
+        }" />
+    </li>
+  </ul>
+</template>
+
+<script>
+import SmartLink from '../smart-link'
+export default {
+  components: {
+    SmartLink
+  },
+  props: {
+    items: {
+      type: Array,
+      required: true,
+    }
+  }
+}
+</script>
+
+<style>
```

Comment:
This style block and the class names on the `ul` and `li` are currently not needed.